### ISSUE TITLE
Couldn't run the triggered component second time in the loop

### ISSUE THREAD

**Issue by VigneshS93** [arduino/Arduino#7347]
I am using arduino to trigger a projector which has 3 frames. I am trying to run each frame twice in one arduino period but the frame is displayed only once. It doesn't appear the second time. Can you please tell me the changes that I have to make in the arduino code so that my projector will project the same frame once again in the loop.

**Comment by per1234** [comment]
This issue tracker is only to be used to report bugs or feature requests. This topic is more appropriate for the Arduino Forum. I'm sure we'll be able to help you with your problem over there.

Please do this:
1. Read http://forum.arduino.cc/index.php?topic=148850
2. If you haven't already done so, create a new thread in the appropriate section of the forum (http://forum.arduino.cc/), following all the rules. Be sure to post your code, using code tags (`</>` button on the toolbar).

**Comment by VigneshS93** [comment]
thank you
